src/main/java/com/ft/methodearticlemapper/transformation/StructuredMethodeSourcedBodyXMLEventHandlerRegistryInnerTable.java

```diff
@@ -28,12 +28,15 @@ public StructuredMethodeSourcedBodyXMLEventHandlerRegistryInnerTable(MethodeBody
 		this.registerStartAndEndElementEventHandlerWrapper(new StripElementAndContentsXMLEventHandler(), "videoPlayer", "web-pull-quote", "plainHtml", "web-background-news", "promo-box");
 
 		this.registerStartAndEndElementEventHandlerWrapper(new RetainWithSpecificAttributesXMLEventHandler("colspan", "rowspan"),
-				"th", "td");
+				"td");
 
 		this.registerStartAndEndElementEventHandlerWrapper(new RetainWithoutAttributesXMLEventHandler(),
 				"thead", "tbody",
 				"tfoot", "caption", "tr");
```

Comment:
`caption` can contain the attribute `data-heading-level`. This attribute was added by Jill to the document; it might be a good idea to consult Luke if he wants this attribute.